components/html-block/html-block.js

```diff
@@ -155,15 +171,53 @@ class HtmlBlock extends LitElement {
 
 		this.shadowRoot.innerHTML += '<div class="d2l-html-block-rendered"></div><slot></slot>';
 
+		this.shadowRoot.querySelector('slot').addEventListener('slotchange', async e => {
+
+			const template = e.target.assignedNodes({ flatten: true })
+				.find(node => (node.nodeType === Node.ELEMENT_NODE && node.tagName === 'TEMPLATE'));
+
+			this._stamp(template);
+
+		});
+		this._renderContainer = this.shadowRoot.querySelector('.d2l-html-block-rendered');
+		this._context = {...this._contextObserverController.values};
+	}
+
+	updated() {
+		super.updated();
+		if (this._contextObjectHasChanged()) {
+			const template = this.querySelector('template');
+			this._stamp(template);
+		}
+	}
+
+	static get _renderers() {
```

Comment:
Do we need to wrap this function call?

Reply:
What do you mean?

Reply:
Could the callers of this just call `getRenderers` directly?

Reply:
Good question. This is existing code (just moved), but I'm not sure if there's a reason to expose this as a static getter. I feel like nothing else needs to use it, so calling the function directly seems perfectly fine to me.